skbase/lookup/_lookup.py

```diff
@@ -556,20 +613,36 @@ def get_package_metadata(
         - "contains_base_objects": whether any module classes that
           inherit from ``BaseObject``.
     """
+    # If someone changed deprecated parameter from the default we'll use it but warn
+    if exclude_nonpublic_modules is not True:
```

Comment:
better maybe: `if not exclude_nonpublic_modules`?

Reply:
Yep, more pythonic. Will push commit.